source/_integrations/twinkly.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: Twinkly
+description: Instructions on how to integrate Twinkly led string to Home Assistant.
+ha_category:
+  - Light
+ha_release: '0.117'
+ha_config_flow: true
+ha_domain: twinkly
+ha_iot_class: Local pull
+---
+
+The `twinkly` integration allows you to control [Twinkly](https://twinkly.com/) LED string from Home Assistant.
+
+The Twinkly devices does not store the effects locally, they are instead re-sent from the Twinkly application each time.
+This means that this integration does not support to change the LED string effect.
+It only supports to configure the brightness and to turn the device on and off.
+
+## Setup
+
+You can setup this integration from the Home Assistant user interface,
```

Comment:
We should describe here how to do that, e.g., by providing a click path and/or step by step instructions.